README.md

```diff
@@ -118,16 +120,9 @@ set -gF window-status-format "#[bg=#{@ctp_surface_1},fg=#{@ctp_fg}] ##I ##T "
 set -gF window-status-current-format "#[bg=#{@ctp_mauve},fg=#{@ctp_crust}] ##I ##T "
 ```
 
-### Upgrading from v0.3.0
-
-If you are upgrading from 0.3.0 to any later versions, please note the following changes:
-
-1. The `status-left` and `status-right` options are no longer controlled by this plugin.
-   You can adjust these manually. See the section on status line modules for more details.
-1. Any custom status line modules may no longer work correctly, and will need to be rewritten
-   to maintain compatibility with newer plugin versions.
+### Upgrading from 0.3
 
-The plugin can be pinned to v0.3.0 if desired: `set -g @plugin 'catppuccin/tmux#v0.3.0'`.
+Breaking changes have been introduced since 0.3, to understand how to migrate your configuration, see pinned issue #291.
```

Comment:
Tiny nitpick from me, just making sure that a link is created for the pinned issue.

```suggestion
Breaking changes have been introduced since 0.3, to understand how to migrate your configuration, see pinned issue [#291](https://github.com/catppuccin/tmux/issues/291).
```